dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -102,9 +102,10 @@ class Catalog extends React.Component<ICatalogProps, ICatalogState> {
     const filteredCharts = this.filteredCharts(allItems);
     const filteredCSVs = this.shouldRenderOperators() ? this.filteredCSVs(csvs) : [];
     const catalogItems = this.getCatalogItems(filteredCharts, filteredCSVs);
-    const items = catalogItems.map(c => (
-      <CatalogItem key={`${c.type}/${c.repo?.name || c.csv}/${c.name}`} item={c} />
-    ));
+    const items = catalogItems.map(c => {
+      const keyComponent = c.type === "operator" ? (c.item as IOperatorCatalogItem).csv : (c.item as IChartCatalogItem).repo.name;
+      return <CatalogItem type={c.type} key={`${c.type}/${keyComponent}/${c.item.name}`} item={c.item} />
```

Comment:
this seems unnecessary complex for just the key but I don't have a better proposal :thinking:

Reply:
It's just for the required unique key, I think? so we could instead just use the index... I wasn't sure as I was only refactoring this code, so left it as it was - let me know if there's a reason for the more structured key?